packages/ethereum/deployments/master-goerli/goerli/HoprChannels.json

```diff
@@ -653,7 +653,11 @@
     "0xa3C8f4044b30Fb3071F5b3b02913DE524F1041dc",
     300
   ],
+<<<<<<< HEAD
```

Comment:
There is a merge-conflict

Reply:
Good catch, an oversight. Fixed in `27deaff`